CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ and this project adheres to
 
 - 🧑‍💻(frontend) change literal section open source #702
 - ♻️(frontend) replace cors proxy for export #695
+- 📝(readme) remove preprod account from readme #812
```

Comment:
I wouldn't list it in the changelog

Reply:
I think that if I don't it breaks the CI. Doesn't it @AntoLC ?

Reply:
No, changelog entry are not mandatory.

Reply:
Fixed